spec/unit/libraries/splunk_password_spec.rb

```diff
@@ -0,0 +1,45 @@
+# coding: UTF-8
+
+require_relative '../spec_helper'
+require 'splunk_password'
+require 'rc4'
+
+describe 'CernerSplunk::splunk_password' do
+  let(:splunk_secret) { 'this_is_the_splunk_secret_key' }
+  let(:plain_text_password) { 'password' }
+  subject { CernerSplunk.splunk_encrypt_password(plain_text_password, splunk_secret) }
+
+  context 'when string to be encrypted is nil or empty' do
```

Comment:
When testing a library with rspec, it is cleaner and easier if you use a `describe` block for each function you want to test. This would look something like 
```
describe 'CernerSplunk::splunk_password' do
  let(:splunk_secret) { 'this_is_the_splunk_secret_key' }
  let(:plain_text_password) { 'password' }
  
  describe '.splunk_encrypt_password' do
    subject { CernerSplunk.splunk_encrypt_password(plain_text_password, splunk_secret) }
    ...
  end
  describe '.splunk_decrypt_password' do
    subject { CernerSplunk.splunk_decrypt_password(plain_text_password, splunk_secret) }
    ...
  end
end
```